recipe_scrapers/_abstract.py

```diff
@@ -19,23 +20,37 @@ class Decorators:
         Define decorators for AbstractScraper methods here.
         """
         @staticmethod
-        def schema_org_priority(function):
+        def schema_org_priority(decorated):
             """
             Use SchemaOrg parser with priority (if there's data in it)
             On exception raised - continue by default.
             If there's no data (no schema implemented on the site) - continue by default
             """
             def schema_org_priority_wrapper(self, *args, **kwargs):
-                if self.schema.data:
-                    try:
-                        return self.schema.__getattribute__(
```

Comment:
@hhursev A small note that I figured that this might be a little more readable with a [`getattr` call](https://github.com/hhursev/recipe-scrapers/pull/144/files#diff-f841370ef246c2b14fd695410f6c9590R30), and that we could check for programmer error at the same time (if someone tries to add a decorator to a function that doesn't exist in the schema)